docs/whats-new.mdx

```diff
@@ -14,6 +14,13 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## March 2024
 
+- 2024-03-13 - [ngrok-go 1.9.1 Released](https://github.com/ngrok/ngrok-go/releases/tag/v1.9.1).
```

Comment:
Can we add the changelog entry? 
https://github.com/ngrok/ngrok-go/blob/v1.9.1/CHANGELOG.md

Reply:
```suggestion
- 2024-03-13 - [ngrok-go 1.9.1 Released](https://github.com/ngrok/ngrok-go/releases/tag/v1.9.1). [Changelog](https://github.com/ngrok/ngrok-go/blob/v1.9.1/CHANGELOG.md)
```